nav2_smac_planner/include/nav2_smac_planner/node_lattice.hpp

```diff
@@ -25,6 +25,7 @@
 #include <utility>
 #include <limits>
 #include <string>
+#include <nlohmann/json.hpp>
```

Comment:
Please add this as a rosdep key to package.xml so that it is installed in the package manager

Reply:
Done